aks-kubesphere-linux/azuredeploy.json

```diff
@@ -0,0 +1,506 @@
+        },
+        "enablePrivateCluster": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "Enable private network access to the kubernetes cluster."
+            }
+        },
+        "ClientResourceName": {
+            "type": "string",
+            "defaultValue": "aksclient",
+            "metadata": {
+                "description": "The name of AKS client node"
+            }
+        },
+        "ClientNodeNetworkInterfaceName": {
+            "type": "string",
+            "defaultValue": "ksnode-nic",
+            "metadata": {
+                "description": "The name of new or existing client node network interface."
+            }
+        },
+        "ClientNodeEnableAcceleratedNetworking": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "Enable client node network accelerated."
+            }
+        },
+        "ClientosDiskType": {
+            "type": "string",
+            "allowedValues": [
+                "StandardSSD_LRS",
+                "Premium_LRS"
+            ],
+            "defaultValue": "StandardSSD_LRS",
+            "metadata": {
+                "description": "Client os Disk type"
+            }
+        },
+        "adminUsername": {
+            "type": "string",
+            "defaultValue": "aksuser",
+            "metadata": {
+                "description": "Client node admin user name."
+            }
+        },
+        "adminPassword": {
+            "type": "string",
+            "defaultValue": "ABCabc321",
+            "metadata": {
+                "description": "Client node admin user password."
+            }
+        },
+        "CloudName": {
+            "type": "string",
+            "defaultValue": "AzureGlobalCloud",
+            "allowedValues": [
+                "AzureChinaCloud",
+                "AzureGlobalCloud"
+            ],
+            "metadata": {
+                "description": "AzureChinaCloud users should change it."
+            }
+        },
+        "SPName": {
+            "type": "string",
+            "metadata": {
+                "description": "Service Principal User Name."
+            }
+        },
+        "SPPassword": {
+            "type": "string",
+            "metadata": {
+                "description": "Service Principal User Password."
+            }
+        },
+        "SPTenant": {
+            "type": "string",
+            "metadata": {
+                "description": "Service Principal User Tenant."
+            }
+        },
+        "SubscriptionID": {
+            "type": "string",
+            "metadata": {
+                "description": "SubscriptionID."
+            }
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/aks-kubesphere-linux/",
```

Comment:
the latest information I have suggests that https://raw.githubusercontent.com was not accessible from China - has this changed?

Reply:
Most Chinese users cannot access git, but Azure Cloud in China can be accessed smoothly. We have carefully checked.